src/main/java/org/assertj/core/internal/Iterables.java

```diff
@@ -917,6 +919,18 @@ public void assertContainsExactly(AssertionInfo info, Iterable<?> actual, Object
                            shouldContainExactly(actual, values, diff.missing, diff.unexpected, comparisonStrategy));
   }
 
+  public void containsAnyOf(AssertionInfo info, Iterable<?> actual, Object[] values) {
+    checkIsNotNull(values);
+    assertNotNull(info, actual);
+    Set<Object> valuesToSearchFor = newTreeSet(values);
+    for (Object element : actual) {
+      if (valuesToSearchFor.contains(element)) {
```

Comment:
if possible  in your IDE, remove curly braces and put `return;` in the same line